app/controllers/your-psp/post-flex.controller.test.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+
+const gatewayAccountFixtures = require('../../../test/fixtures/gateway-account.fixtures')
+
+describe('Post 3DS Flex controller', () => {
+  const gatewayAccountExternalId = 'a-gateway-account-external-id'
+  const credentialId = 'a-valid-credential-id'
+  let req
+  let res
+  let next
+  let postCheckWorldpay3dsFlexCredentials
+  let post3dsFlexAccountCredentials 
+  let updateIntegrationVersion3dsMock
+  let renderErrorViewMock
+
+  beforeEach(() => {
+    req = {
+      params: { credentialId },
+      flash: sinon.spy(),
+      body: {
+        'organisational-unit-id': '111111111111111111111111',
+        'issuer': '111111111111111111111111',
+        'jwt-mac-key': '11111111-1111-1111-1111-111111111111'
+      }
+    }
+    res = {
+      setHeader: sinon.stub(),
+      status: sinon.spy(),
+      redirect: sinon.spy(),
+      render: sinon.spy()
+    }
+    next = sinon.spy()
+  })
+
+  it('when `account.type=live`, then should toggle 3DS Flex on by setting 3DS integration version to 2', async () => {
```

Comment:
Can it be started with `should` so it reads `it should ...`. Maybe below?

```suggestion 
  it('should set 3DS integration version to 2 for LIVE account', async () => {
```